### PR TITLE
Add: add document for rewrite test

### PR DESCRIPTION
**Pull request by taojintianxia**

### CODE REVIEW

**Comment by taojintianxia** [comment]
@terrymanu please review the fix.

File: document/current/content/features/test-engine/rewrite-test-engine.cn.md [hunk 1]
```diff
@@ -5,4 +5,62 @@ title = "SQL改写测试引擎"
 weight = 4
 +++
 
-待完成
+## 目标
+
+面向逻辑库与逻辑表书写的SQL，并不能够直接在真实的数据库中执行，SQL改写用于将逻辑SQL改写为在真实数据库中可以正确执行的SQL。 它包括正确性改写和优化改写两部分，所以 rewrite 的测试都是基于这些改写方向进行校验的。
```
Comment (terrymanu) [inline document/current/content/features/test-engine/rewrite-test-engine.cn.md]: Should use Chinese word to keep it consistent

File: document/current/content/features/test-engine/rewrite-test-engine.en.md [hunk 2]
```diff
@@ -5,4 +5,65 @@ title = "SQL Rewrite Test Engine"
...
+        <input sql="SELECT * FROM sharding_db.t_account WHERE amount = ?" parameters="1000" />
+        <output sql="SELECT * FROM t_account_0 WHERE amount = ?" parameters="1000" />
+        <output sql="SELECT * FROM t_account_1 WHERE amount = ?" parameters="1000" />
+    </rewrite-assertion>
+</rewrite-assertions>
+```
+
+After set up the assert data and environment configuration, rewrite test engine will assert the corresponding SQL without any Java code modification.
+
+
```
Comment (terrymanu) [inline document/current/content/features/test-engine/rewrite-test-engine.en.md]: Please useless blank line

File: document/current/content/features/test-engine/rewrite-test-engine.cn.md (hunk 1 above)
Comment (taojintianxia) [reply document/current/content/features/test-engine/rewrite-test-engine.cn.md]: got it

**Comment by taojintianxia** [comment]
please review the changes

**Pull request merged**